feature2/another-feature.txt

```diff
@@ -0,0 +1,3 @@
+This file was added to feature2 branch
+
+Another line was added
```

Comment:
It was a good idea to add this line here

Reply:
Thanks for your comment